internal/compiler/passes/embed_images.rs

```diff
@@ -88,7 +88,9 @@ fn embed_image(
             if let Some(file) = crate::fileaccess::load_file(std::path::Path::new(path)) {
                 let mut kind = EmbeddedResourcesKind::RawData;
                 #[cfg(not(target_arch = "wasm32"))]
-                if std::env::var("SLINT_PROCESS_IMAGES").is_ok() {
+                if std::env::var("SLINT_PROCESS_IMAGES").is_ok()
+                    || std::env::var("DEP_I_SLINT_BACKEND_MCU_PROCESS_IMAGES").is_ok()
```

Comment:
A more generic way would perhaps be to add this to `CompilerConfiguration` (like `scale_factor`). Then perhaps the processing of `DEP_I_SLINT_BACKEND_MCU_*` can be limited to `api/rs/build/lib.rs`.

That also paves a way for the project to override this in the future. For example for some projects it might make sense to delegate all image loading off an SD-card, or that instead of embedding glyphs the true type font should be embedded in flash (to have access to shaping tables).

Either way, it makes a lot of sense for the mcu backend to provide the defaults the way this patch does :)

Reply:
I agree

Reply:
But that doesn't work when using the `slint!` macro then :-/

Reply:
> But that doesn't work when using the slint! macro then :-/

Yes, neither does any of the build system stuff, like setting the memory map file for the linker. Although I guess for that users could use `build.rs` with just the `print_rustc_flags()` call and continue to use the macro in `.rs` files.